### ISSUE TITLE
NuGet package is more than 3 years out of date and does not include Persona support (which was introduced 3 years ago).

### ISSUE THREAD

**Issue by BrainSlugs83** [OfficeDev/ews-managed-api#169]
The nuget package linked to in the readme.md appears to have been last published over 3 years ago, on Jan, 15, 2015.

And Personas were added to the API in March of 2015.

Please publish an update to the NuGet package that is linked to in the readme.md, so that this feature will be available to NuGet consumers of the managed API.

(If the latest code is not ready for Production usage, you can still publish the update as a pre-release version that users have to specifically opt. in to using. -- But even that's better than having the features be completely unavailable.)

**Comment by BrainSlugs83** [comment]
Or perhaps is this the latest package? https://www.nuget.org/packages/Microsoft.Exchange.WebServices.NetStandard/

If so, then the link in the readme.md needs to be updated.

**Comment by poizan42** [comment]
@BrainSlugs83 No, that is a fork maintained by @sherlock1982. Unfortunately it does not seem that msft are dedicating any resources to this anymore.

**Comment by sherlock1982** [comment]
Yes I try to maintain it cause we use it internally. Changed to Tasks, added some fixes for Linux e.t.c.

**Comment by BrainSlugs83** [comment]
Well, the NuGet package that's linked in this repo's `readme.md` says the author is "Microsoft" -- so, whoever maintains this repo is aware of it.  And they should either remove the link to it, or update their NuGet package. 😑

## Readme.md:
![readme.md screenshot](https://user-images.githubusercontent.com/5217366/62391820-b6133500-b51a-11e9-9834-6da954522e72.png)


## Nuget.org:
![Nuget.org screenshot](https://user-images.githubusercontent.com/5217366/62391907-e529a680-b51a-11e9-9ba5-08709d685eed.png)
